### PR TITLE
ign -> gz Macro Migration : gz-fuel-tools

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/737

### CODE REVIEW

File: src/ign.hh [hunk 1]
```diff
@@ -22,7 +22,8 @@
 
 /// \brief External hook to read the library version.
 /// \return C-string representing the version. Ex.: 0.1.2
-extern "C" GZ_FUEL_TOOLS_VISIBLE char *ignitionVersion();
+extern "C" GZ_FUEL_TOOLS_VISIBLE char *gzVersion();
+extern "C" GZ_FUEL_TOOLS_VISIBLE char GZ_DEPRECATED(8) *ignitionVersion();
```
Comment (chapulina) [inline src/ign.hh:26]: No need to tick-tock, this isn't installed

File: Migration.md [hunk 2]
```diff
@@ -6,6 +6,13 @@
 * **Replacement**: `gz` namespace
 * **Deprecation**: `ignition/...` headers
 * **Replacement**: `gz/...` headers
+* **Deprecation**: `ignitionVersion`
+* **Replacement**: `gzVersion()`
```
Comment (chapulina) [inline Migration.md:10]: This can be removed, no need to tick-tock the private function

**Review by chapulina — CHANGES_REQUESTED** [review]

File: src/ign.hh (hunk 1 above)
Comment (methylDragon) [reply src/ign.hh]: https://github.com/gazebosim/gz-fuel-tools/pull/259/commits/59add5051848659b9c291109fafe3b429868b612

File: Migration.md (hunk 2 above)
Comment (methylDragon) [reply Migration.md]: https://github.com/gazebosim/gz-fuel-tools/pull/259/commits/59add5051848659b9c291109fafe3b429868b612

**Comment by chapulina** [comment]
@osrf-jenkins run tests please!

**Comment by chapulina** [comment]
No new test failures

**Pull request merged**